<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastFMImportDialog</class>
 <widget class="QDialog" name="LastFMImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import data from last.fm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page_start">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_start_top">
         <property name="text">
          <string>Choose data to import from last.fm</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_last_played">
         <property name="text">
          <string>Last played</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_playcounts">
         <property name="text">
          <string>Play counts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_start_bottom">
         <property name="text">
          <string>Warning: Play counts and last played from last.fm will completely replace the same data for the matched songs. Play counts will replace the data based on artist and song title for the same albums! Please backup your database before you start.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_progress">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_progress_top">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressbar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_progress_bottom">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_buttons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_go">
       <property name="text">
        <string>Go!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
